ibm/structures.go

```diff
@@ -51,6 +51,8 @@ const (
 	ResourceName = "resource_name"
 	//ResourceCRN ...
 	ResourceCRN = "resource_crn"
+	//ResourceCheckSum
+	ResourceCheckSum = "resource_checksum"
```

Comment:
don't add ResourceCheckSum here...it should be part of the resource/datasource..

The ResourceName,ResourceCRN  ...are some generic arguments added for multiple resources